tests/openwisp2/sample_notifications/migrations/0003_default_notification_setting.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.0.8 on 2020-08-04 20:50
```

Comment:
in order to keep things sane for us, we are trying to avoid generating migrations for the sample app, so please edit the first migration